BinaryObjectScanner/Packer/SevenZipSFX.cs

```diff
@@ -46,8 +53,64 @@ public class SevenZipSFX : IExtractablePortableExecutable, IPortableExecutableCh
 
         /// <inheritdoc/>
         public string? Extract(string file, PortableExecutable pex, bool includeDebug)
+            => Extract(file, includeDebug);
+
+        /// <inheritdoc/>
+        public string? Extract(string file, bool includeDebug)
```

Comment:
Are you expecting to include NE extraction at some point? If not, then this separation is unnecessary.